frontend/src/bundles/common/components/stepper/components/step-icon/step-icon.tsx

```diff
@@ -0,0 +1,58 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { type ValueOf } from 'shared';
+
+import { Box, Icon } from '~/bundles/common/components/components.js';
+import { IconName } from '~/bundles/common/icons/icons.js';
+
+import { Step } from '../../enums/enums.js';
+
+type Properties = {
+    step: ValueOf<typeof Step>;
+};
+
+const StepIcon: React.FC<Properties> = ({ step }) => {
+    if (step === Step.COMPLETED) {
+        return (
+            <Box>
```

Comment:
Do we need an additional Box wrapper here?
The same for components below

Reply:
Yes, I tried removing it, but then the before and after CSS styles didn't work properly.